starlette_zipkin/middleware.py

```diff
@@ -29,9 +35,14 @@ def __init__(
     async def dispatch(
         self, request: Request, call_next: RequestResponseEndpoint
     ) -> Response:
-        tracer = get_tracer()
-        if tracer is None:
-            tracer = await init_tracer(self.config)
+        # check for externally provided tracer - if none provided, we will create one and hold on to it to reuse it,
+        # similar to https://github.com/aio-libs/aiozipkin/blob/master/examples/aiohttp_example.py
+        if (external_tracer := get_tracer()) is None:
```

Comment:
You will break compatibility with python3.7 here. (walrus operator is python3.8)

Reply:
Ah, good point, no need to do that obv, let me adjsut